packages/perseus/src/types.ts

```diff
@@ -544,6 +545,7 @@ export type WidgetScorerFunction = (
  * A union type of all the functions that provide public widget options.
  */
 export type PublicWidgetOptionsFunction =
+    | typeof getNumericInputPublicWidgetOptions
```

Comment:
Is this file the best place for this union? It is used to type the `getPublicWidgetOptionsFunction` in the `widgets.ts` file, which was created to be the function used in the editor to access the correct `getPublicWidgetOptions` function.

Reply:
I think this is fine!